dynamiq/nodes/tools/e2b_sandbox.py

```diff
@@ -68,14 +64,14 @@ class E2BInterpreterTool(ConnectionNode):
         _sandbox (Optional[Sandbox]): Persistent sandbox instance (if enabled).
     """
 
-    group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
-    name: str = "code-interpreter_e2b"
+    name: str = "code-interpreter-e2b"
     description: str = DESCRIPTION
     connection: E2BConnection
     installed_packages: list = []
     files: list[tuple[str | bytes, str]] | None = None
     persistent_sandbox: bool = True
     _sandbox: Sandbox | None = None
+    input_schema: type[E2BInterpreterInputSchema] = E2BInterpreterInputSchema
```

Comment:
I think such fields should be private. `_input_schema`?

Reply:
I guess user might want to get this schema, to do some validation or whatever.
@acoola ?

Reply:
Maybe we need to keep it private, but use public getter and setter methods
```
 _input_schema: Type[E2BInterpreterInputSchema] = E2BInterpreterInputSchema
 
    @property
    def input_schema(self) -> Type[E2BInterpreterInputSchema]:
        """
        Provides access to the input schema.
        """
        return self._input_schema

    @input_schema.setter
    def input_schema(self, schema: Type[E2BInterpreterInputSchema]):
        """
        Sets a new input schema, allowing flexibility in validation needs.
        """
        if not isinstance(schema, type):
            raise TypeError("input_schema must be a type.")
        #validation can be also here
        self._input_schema = schema
```

Reply:
We don't need getter and setter. This schema should be as Constant, so try to use ClassVar to define this.
So user should be able to get node schema without initialization too:
```
schema = E2BInterpreterTool.input_schema
tool = E2BInterpreterTool(...)
assert schema == tool.input_schema
```